modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/PublishEngageWorkflowOperationHandler.java

```diff
@@ -213,6 +214,9 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, Jo
     boolean checkAvailability = option(op.getConfiguration(CHECK_AVAILABILITY)).bind(trimToNone).map(toBool)
             .getOrElse(true);
 
+    boolean addData = option(op.getConfiguration(LIST_ELEMENTS)).bind(trimToNone).map(toBool)
```

Comment:
The variable name is pretty much meaningless. What about using:

```suggestion
    boolean publichElements = option(op.getConfiguration(LIST_ELEMENTS)).bind(trimToNone).map(toBool)
```

Reply:
`publich -> publish`

Reply:
`publich -> publish`